vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -613,6 +666,28 @@ bool ChakraRuntime::instanceOf(const facebook::jsi::Object &obj, const facebook:
 
 #pragma endregion Functions_inherited_from_Runtime
 
+// Rewrite some error messages
+void ChakraRuntime::RewriteErrorMessage(JsValueRef jsError) {
+  JsPropertyIdRef messagePropertyId;
+  VerifyChakraErrorElseThrow(JsGetPropertyIdFromName(L"message", &messagePropertyId));
+  JsValueRef message;
+  VerifyChakraErrorElseThrow(JsGetProperty(jsError, messagePropertyId, &message));
+  JsValueType valueType;
+  VerifyChakraErrorElseThrow(JsGetValueType(message, &valueType));
+  if (valueType == JsValueType::JsString) {
+    wchar_t const *messageStr;
+    size_t messageLength;
+    VerifyChakraErrorElseThrow(JsStringToPointer(message, &messageStr, &messageLength));
```

Comment:
>messageStr [](start = 59, length = 10)

(nit) just curious, what is the lifetime of this char* array? while "message" jsString is alive?

Reply:
Yes, it is alive while the corresponding JsValueRef is alive. In this case it is the 'message' variable.

---
In reply to: [460192667](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r460192667) [](ancestors = 460192667)